cnxdb/tests/cli/test_main.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+
+import psycopg2
+import pytest
+
+from .. import testing
+
+
+def _translate_parts_to_args(parts):
+    """Translates connection string parts to arguments"""
+    return ['-h', parts['host'],
+            '-p', parts['port'],
+            '-d', parts['dbname'],
+            '-U', parts['user'],
+            ]
+
+
+def test_init(connection_string_parts, db_wipe):
```

Comment:
It's possible to use something like this:

```
pytestmark = pytest.mark.usefixtures('db_wipe')
```

or group the tests into:

```
@pytest.mark.usefixtures('db_wipe')  # or @pytest.mark.usefixtures('db_init', 'db_wipe') for testing venv
class InitTestCase:
    def test_called_twice(connection_string_parts):
        pass
```

I think it's nicer than having arguments that are not used.

As for `connection_string_parts`, isn't it better to import it at the top of the file and use it in all the tests rather than having it as a fixture?